lib/domain/app_state/room/chat_get_pinned_events_state.dart

```diff
@@ -29,6 +29,11 @@ class ChatGetPinnedEventsNoResult extends Failure {
   List<Object?> get props => [];
 }
 
+class ChatGetPinnedEventsRoomIsNull extends Failure {
```

Comment:
`GetPinnedEventsRoomEmptyFailure`

Reply:
not empty, it's null

Reply:
`GetPinnedEventsRoomIsNullFailure`

Reply:
?

Reply:
IMO: `ChatGetPinnedEventsRoomIsNull` extends Failure is enough

Reply:
```suggestion
class CannotGetPinnedMessages extends Failure {
```